source/reference/access-mdb-shell-help.txt

```diff
@@ -0,0 +1,241 @@
+.. _mdb-shell-help:
+
+===========================
+Access the |mdb-shell| Help
+===========================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+``mongosh`` provides additional help information in its "online"
```

Comment:
Why is "online" in quotes?

Reply:
It's this way in the [manual](https://docs.mongodb.com/manual/tutorial/access-mongo-shell-help/#access-the-mongo-shell-help). think it's meant to emphasize it's not "online" -- some people call any type of help you access "online help".

fixed here.